packages/typespec-client-generator-core/test/packages/parameters.test.ts

```diff
@@ -0,0 +1,1005 @@
+import { AzureCoreTestLibrary } from "@azure-tools/typespec-azure-core/testing";
+import { deepStrictEqual, ok, strictEqual } from "assert";
+import { beforeEach, describe, it } from "vitest";
+import {
+  SdkHeaderParameter,
+  SdkHttpOperation,
+  SdkPathParameter,
+  SdkQueryParameter,
+  SdkServiceMethod,
+} from "../../src/interfaces.js";
+import { SdkTestRunner, createSdkTestRunner } from "../test-host.js";
+import { getServiceMethodOfClient, getServiceWithDefaultApiVersion } from "./utils.js";
+
+describe("typespec-client-generator-core: parameters", () => {
+  let runner: SdkTestRunner;
+
+  beforeEach(async () => {
+    runner = await createSdkTestRunner({ emitterName: "@azure-tools/typespec-python" });
+  });
+
+  it("path basic", async () => {
```

Comment:
Seems most of tests are moved from `package.test.ts`, could you please highlight the part of tests you added?